pkg/services/authn/clients/ext_jwt.go

```diff
@@ -46,68 +55,97 @@ type ExtendedJWT struct {
 	log         log.Logger
 	userService user.Service
 	signingKeys signingkeys.Service
+	verifier    authlib.Verifier[ExtendedJWTClaims]
 }
 
 type ExtendedJWTClaims struct {
 	jwt.Claims
-	ClientID     string              `json:"client_id"`
-	Groups       []string            `json:"groups"`
-	Email        string              `json:"email"`
-	Name         string              `json:"name"`
-	Login        string              `json:"login"`
-	Scopes       []string            `json:"scope"`
-	Entitlements map[string][]string `json:"entitlements"`
+	// Access policy scopes
+	Scopes []string `json:"scopes"`
+	// Grafana roles
+	Permissions []string `json:"permissions"`
```

Comment:
How come we are using RBAC roles here and not just plain permissions as the name suggest?